### ISSUE TITLE
Missing Frames?

### ISSUE THREAD

**Issue by duerig** [ThirdPartyNinjas/GlitchAssets#1]
Great idea on converting them to 2x bitmaps. I notice that on a lot of assets, especially items, there are multiple frames and your scripts seem to have only exported the a single frame. For example:

https://github.com/ThirdPartyNinjas/GlitchAssets/blob/master/Items/tools/pick/fancy_pick/fancy_pick.png

This is the only image of a pick you have and it is a single image of a broken pick. I think the other frames must have images of the normal pick but your script doesn't see them.

**Comment by ThirdPartyNinjas** [comment]
Yep, good catch. I've got three versions of the scripts, and they all do slightly different exporting. I just tried to pick the best one experimentally.

Some of the files (the wardrobes were particularly bad) would put out thousands of duplicate frames, and it was taking hours to export. I'll see if I can clean that up.

**Comment by duerig** [comment]
Thanks very much. It would be even better to get these into .svg format. Especially the wardrobe pieces. Looking through, a lot of them are intended to be colored after the fact and that is something you can do on .svg nodes embedded in html.

It would be awesome to get together the right code to allow people to connect these up into complete customized avatar portraits.

**Comment by ThirdPartyNinjas** [comment]
I don't think I'd be much help in writing a converter to svg. My knowledge of the vector formats is sorely lacking.

I tried my scripts and loaded up the flash file and was able to add one more pick image. There was an error on that file, though. It's trying to load a *.as (actionscript?) file that I can't find.

https://github.com/ThirdPartyNinjas/GlitchAssets/blob/master/Items/tools/pick/fancy_pick/pick_anim.png

I have a feeling there are going to be a bunch of missed frames here and there. There are a ton of input files, and this is the first time I'd ever launched Flash, much less written a script for it. :)
